index.html

```diff
@@ -0,0 +1,281 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Portfolio-Mobile</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <header>
+      <nav class="header">
+        <div class="logo">
+          <a href="#">Welcome</a>
+        </div>
+
+        <div class="vector">
+          <span></span>
+          <span></span>
+          <span></span>
+        </div>
+      </nav>
+    </header>
+
+    <main>
+      <section class="porfolio">
+        <h1>Hey there! I'm Michael</h1>
+
+        <h2>I'm a Software Developer</h2>
+
+        <p>
+          I can help you build a product, feature or website. Look through some
+          of my work and experience! If you like what you see and have a project
+          you need coded, don't hestiate to contact me.
+        </p>
+
+        <ul class="social">
+          <li>
+            <a href="https://github.com/Mike47ip"
+              ><ion-icon name="logo-github"></ion-icon
+            ></a>
+          </li>
+          <li>
+            <a href="https://www.linkedin.com/in/michael-darkwah-81a039141/"
+              ><ion-icon name="logo-linkedin"></ion-icon
+            ></a>
+          </li>
+          <li>
+            <a href="https://twitter.com/MikeDark@47"
+              ><ion-icon name="logo-twitter"></ion-icon
+            ></a>
+          </li>
+          <li>
+            <a href="https://web.facebook.com/Michael"
+              ><ion-icon name="logo-facebook"></ion-icon
+            ></a>
+          </li>
+        </ul>
+
+        <div class="disabled">
+          <img src="./Images/Disabled.png" alt="scroll" />
+        </div>
+      </section>
+
+      <section class="projects">
+        <h2 class="head">My Recent works</h2>
+
+        <div class="span"></div>
+
+        <div class="container">
+          <div class="card">
+            <div class="title">
+              <h2 class="text">Multi-Post Stories Gain+Glory</h2>
+              <ul class="server">
+                <li>Ruby on rails</li>
+                <li>CSS</li>
+                <li>JavaScript</li>
+                <li>HTML</li>
+              </ul>
+              <button type="button" class="button">See Project</button>
+            </div>
+          </div>
+
+          <div class="card">
+            <div class="title">
+              <h2 class="text">Multi-Post Stories Gain+Glory</h2>
+              <ul class="server">
+                <li>Ruby on rails</li>
+                <li>CSS</li>
+                <li>JavaScript</li>
+                <li>HTML</li>
+              </ul>
+              <button type="button" class="button">See Project</button>
```

Comment:
Greate button! it has active stage styles 👍 . But you also need to consider adding hover stage styles. please consider adding hover styles as per in the design. 👍 
![image](https://github.com/Mike47ip/mobilePortfolioWeb/assets/55840999/38feafc2-829d-4e75-9f5d-161c1c82da58)